### PR TITLE
[SPH-TASK]Connecting to MySQL Database

### PR DESCRIPTION
**Pull request by ENRexGeorge**
## Task Link
- https://app.asana.com/0/1202720049437748/1202720060611018/f

## Commands
- 'pip install django' if your `.venv` is not detecting Django
- `cd be`
- `cd compendium`
- `python manage.py runserver`

## Pre-conditions
-The Backend dev should have set the Path of their python locale on the environment variable on their local machine
- The Backend dev should have MySQL installed in their locale
- The Backend dev should have their own `.env ` file, You can always refer to the `env.example` as a template for your `.env`

## Expected Output
- To connect the Django in the Database
- To view the Default Tables added in MySQL 
- To add the Migration Task from DJango in MySQL"

## Notes
-None

## Screenshots
-
image
image
image
image
image

### CODE REVIEW

**Comment by roseaugusto** [comment]
@ENRexGeorge 
please do the following;

1. put pycache in the `.gitignore` so that it won't be pushed in github
2. please consider adding `.env.example` instead of hard coding the credentials in the `settings.py`. Take note that other members may used different values like ports as well. Make sure to allow it to be dynamic.
4. Your task description in asana includes making the migrations from the team's erd.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/be/compendium/compendium/__pycache__
+/be/compendium/compendium/.env
```
Comment (escbooster12-dev) [inline .gitignore]: add newline (apply also in **be/compendium/compendium/.env.example**)

File: be/compendium/compendium/.env.example [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+DATABASE_POST=
+
+
+
+
+
+
+
+
+
```
Comment (escbooster12-dev) [inline be/compendium/compendium/.env.example]: remove extra line

**Comment by BreadOnce** [comment]
I pulled the repo yesterday and I was able to run it on my end.
I think we need to add an extra instruction in running the virtual environment. 

Apart from that, everything looks good to me.

**Pull request merged**